src/main/kotlin/me/modmuss50/mpp/platforms/curseforge/Curseforge.kt

```diff
@@ -53,6 +53,11 @@ interface CurseforgeOptions : PlatformOptions, PlatformOptionsInternal<Curseforg
     @get:Input
     val apiEndpoint: Property<String>
 
+    // Set the type of changelog for CurseForge. If unset, defaults to MARKDOWN
+    @get:Input
+    @get:Optional
```

Comment:
I would set the default value of this using `convention` in the same way `apiEndpoint` is set. That way its not optional, just has a default value.